### ISSUE TITLE
The ConfiguredFeature for exp_ore:block_exp_ore is not registered

### ISSUE THREAD

**Issue by TelepathicGrunt** [Krevik/ExpOreMod#1]
Hello! I was running my mod called Blame with ExpOreMod and it seemed to have found that the Exp Ore ConfiguredFeature is not registered. This can be an issue for mod compatibility as under certain conditions, unregistered ConfiguredFeatures can basically prevent other mod's registered ConfiguredFeatures from spawning if in the same generation stage.

By that I mean, if mod A adds an unregistered CF to the ore generation stage and the biome's codec reaches it first, it will choke and basically nuke mob B's registered CFs afterwards. Here's a case where BetterCaves forgot to register their CF and caused several CFs from Oh The Biomes You'll Go to stop spawning in the world: yungnickyoung/YUNGs-Better-Caves#75

Here's a more detailed explanation of why this happens in the biome's codec:
![image](https://user-images.githubusercontent.com/40846040/95459000-ff41ee80-0940-11eb-8569-52a2be15cc4d.png)

Specifically, when you call .withConfiguration on a Feature, you create a ConfiguredFeature. This is what should be registered to the WorldgenRegisties at mod init (you can do it in FMLCommonSetupEvent so you have your config ready too if it is needed). 

Anyway here's an example from my mod RepurposedStructures of me registering all my ConfiguredFeatures.
https://github.com/TelepathicGrunt/RepurposedStructures/blob/584433a0745338802c84e9f498dc063c1f5505f8/src/main/java/com/telepathicgrunt/repurposedstructures/modinit/RSConfiguredStructures.java#L72-L74

I hope this helps!

**Comment by Krevik** [comment]
Well will take a look into it, however since forge started breaking registries (especially these features ones) I prefer to use working not registred features instead of bugged registred features. What minecraft version did you use?

**Comment by TelepathicGrunt** [comment]
What do you mean? The breaking change forge did a while ago was forcing people to use the forge registry to register anything instead of using the Minecraft registries (if the Forge registry exists). If so, I actually have an example where I register to the forge registries that you can copy too.
https://github.com/TelepathicGrunt/RepurposedStructures/blob/584433a0745338802c84e9f498dc063c1f5505f8/src/main/java/com/telepathicgrunt/repurposedstructures/modinit/RSFeatures.java#L15-L17

Basically, Features has a Forge registry so they should be registered there. The ConfiguredFeatures do not have a Forge Registry which is why we register those to the Minecraft registry. It's a bit confusion yeah but hopefully that makes sense. (Feature and ConfiguredFeature are two different things)

**Comment by TelepathicGrunt** [comment]
Oh and on the minecraft version, I was using 1.16.4 with 35.1.13 forge

**Comment by Krevik** [comment]
> What do you mean? The breaking change forge did a while ago was forcing people to use the forge registry to register anything instead of using the Minecraft registries (if the Forge registry exists). If so, I actually have an example where I register to the forge registries that you can copy too.
> https://github.com/TelepathicGrunt/RepurposedStructures/blob/584433a0745338802c84e9f498dc063c1f5505f8/src/main/java/com/telepathicgrunt/repurposedstructures/modinit/RSFeatures.java#L15-L17
> 
> Basically, Features has a Forge registry so they should be registered there. The ConfiguredFeatures do not have a Forge Registry which is why we register those to the Minecraft registry. It's a bit confusion yeah but hopefully that makes sense. (Feature and ConfiguredFeature are two different things)

Yea they should, but when playing with custom dimension (my main old project) there was no way to register feature and to RUN the mod at the same time, or even if it was possible - the feature was getting totally bugged. So my old habbits here :P Okay thanks gonna register the feature.

**Comment by Krevik** [comment]
Fixed by the latest commit.
